### PR TITLE
Adapted files to comply with QGate5

### PR DESCRIPTION
**Pull request by stephanbcbauer**
Adapted files to comply with QGate5

- [TRG - 1.01](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-5-1) README.md - no valid installation instructions available. Either describe it in README.md or in (missing INSTALL.md)
-  [TRG - 1.02](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-2) INSTALL.md - missing (and required, as REAME.md contains no valid installation instruction)
-  [TRG - 2.03](https://eclipse-tractusx.github.io/docs/release/trg-2/trg-2-3) Repo Structure - Consider add optional file [AUTHORS.md](https://eclipse-tractusx.github.io/docs/release/trg-2/trg-2-3#authorsmd-optional).
-  [TRG - 2.03](https://eclipse-tractusx.github.io/docs/release/trg-2/trg-2-3) Repo Structure - CODE_OF_CONDUCT.md file is missing
-  [TRG - 2.03](https://eclipse-tractusx.github.io/docs/release/trg-2/trg-2-3) Repo Structure - CONTRIBUTING.md file is missing
-  [TRG - 2.03](https://eclipse-tractusx.github.io/docs/release/trg-2/trg-2-3) Repo Structure - DEPENDENCIES file - when did you last check your deps using the [Eclipse Dash tool](https://github.com/eclipse/dash-licenses)? It's 5 months old and contains 31 restricted entries. Does it contain all dependencies of your code? System Team can help opening IP Tickets.
-  [TRG - 2.03](https://eclipse-tractusx.github.io/docs/release/trg-2/trg-2-3) Repo Structure - NOTICE.md is outdated. Do not reference catena-x ng repo in top headline and as source, is 3rd Party content list up2date?
-  [TRG - 4.01](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-1) Image Tagging - Consider using full fledged semver tagging (0, 0.1, 0.1.5, latest, [example](https://github.com/catenax-ng/k8s-helm-example/pkgs/container/k8s-helm-example))
-  [TRG - 4.02](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-2) Base Image - Consider to use temurin as base image for java based applications (we already discussed about it, this remark is not mandatory for release 3).
-  [TRG - 5.02](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-2) Chart Structure - .helmignore missing expected content
-  [TRG - 5.02](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-2) Chart Structure - add missing license file (both charts)
-  [TRG - 5.02](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-2) Chart Structure - Chart Readme.md (control plane) contains reference to catenax-ng.
-  [TRG - 5.05](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-5) Chart Values - Consider using a more precise image section in values.yaml
-  Fix license header in all source code files to match Eclipse recommendations

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,3111 @@
+
```
Comment (Siegfriedk) [inline CODE_OF_CONDUCT.md:1]: I think this file is broken? Its huge and contains html

File: charts/edc-dataplane/LICENSE [hunk 2]
```diff
@@ -0,0 +1,2499 @@
+
+
```
Comment (Siegfriedk) [inline charts/edc-dataplane/LICENSE:2]: This file also looks broken; is that expected?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,3065 @@
+
+
+
```
Comment (carslen) [inline CONTRIBUTING.md:3]: Do not use HTML in markdown files. There are example files you can simply copy and paste.

File: charts/edc-controlplane/LICENSE [hunk 4]
```diff
@@ -0,0 +1,2499 @@
+
+
+
```
Comment (carslen) [inline charts/edc-controlplane/LICENSE:3]: Do not use HTML in markdown files. There are example files you can simply copy and paste.

File: charts/edc-dataplane/LICENSE [hunk 5]
```diff
@@ -0,0 +1,2499 @@
...
+
+
+
+
```
Comment (carslen) [inline charts/edc-dataplane/LICENSE:6]: Do not use HTML in markdown files. There are example files you can simply copy and paste.

**Review by carslen — CHANGES_REQUESTED** [review]
Please see comments.

**Comment by stephanbcbauer** [comment]
@Siegfriedk @carslen Sorry for the mess adapted the files

**Review by carslen — APPROVED** [review]
LGTM

**Comment by ndr-brt** [comment]
@stephanbcbauer thanks, does this need to be backported to product-edc?

**Pull request merged**